core/dbt/contracts/graph/model_config.py

```diff
@@ -435,6 +435,7 @@ class NodeConfig(NodeAndTestConfig):
         default_factory=dict,
         metadata=MergeBehavior.Update.meta(),
     )
+    delimiter: str = ","
```

Comment:
This should go in [`SeedConfig`](https://github.com/dbt-labs/dbt-core/blob/main/core/dbt/contracts/graph/model_config.py#L500), now defined separately, rather than `NodeConfig`